lib/containers/urls.pm

```diff
@@ -156,7 +156,8 @@ sub get_3rd_party_images {
     my @images = (
         "registry.opensuse.org/opensuse/leap",
         "registry.opensuse.org/opensuse/tumbleweed",
-        "ghcr.io/linuxcontainers/alpine:latest"
+        "ghcr.io/linuxcontainers/alpine:latest",
+        "quay.io/centos/centos:stream9"
```

Comment:
Same here, let's use `docker.io` links for the registry.

Reply:
Fixed.  CentOS images in DockerHub are deprecated: https://hub.docker.com/_/centos